Added Column Classes Panel

ben-schwen opened the pull request:
Hey Stan,

I added a new panel to let the user manually select his column classes out of a dropdown menu. Since I also wanted to bound these values, I had to use 'rhandsontable'. 

cheers,
Ben

ben-schwen committed:
Added column types

Added a new tab panel to allow the user to set the column types manually.

updated Readme.md

fixed Read.R

Update GREA.R

Updated dependencies

Fixed check()

Added some error coding for developing under windows. Added used functions to namespace.

fixed styling

Merge pull request #23 from wiep/master

Apply `make.names` to the suggested variable name

ben-schwen committed:
Added column types

Added a new tab panel to allow the user to set the column types manually.

ben-schwen committed:
updated Readme.md

ben-schwen committed:
fixed Read.R

ben-schwen committed:
Update GREA.R

ben-schwen committed:
Updated dependencies

ben-schwen committed:
Fixed check()

Added some error coding for developing under windows. Added used functions to namespace.

ben-schwen committed:
fixed styling

Stan125 commented:
Hey Ben,

nice job. Good idea. There are two small points remaining, however:
- It's not possible to do vertical scrolling on the data types `handsontable`. Horizontal Scrolling works
- There seems to be a bug with making the correct R code for display on the console. Try importing `inst/extdata/excel_aachen.xlsx` and leaving all column classes as is. Then the following code is pasted into the console:
![Screenshot 2019-07-12 at 16 29 01](https://user-images.githubusercontent.com/17404160/61135964-facf2180-a4c2-11e9-81f4-42f80c38d420.png)

Best,
Stani

ben-schwen committed:
Fixed column types

Changed column types from gsub insert to be more consistent with Stans style
Added isolation to hot_table

ben-schwen committed:
fixed merge accident

ben-schwen committed:
Merge branch 'master' into column-Types

ben-schwen committed:
Merge pull request #1 from ben-schwen/column-Types

Column types

ben-schwen committed:
Added Reload button

ben-schwen committed:
Merge remote-tracking branch 'origin/master'

ben-schwen committed:
fixed merge

ben-schwen commented:
Hey Stani,

I reworked the code to be more consistent with your style. Now I add an additional argument to the read function instead of "hacking" the expr string on the end.

The two points you mentioned are cleared. 

I also added a "Reload" button to the columns tab to give the possibility to reload the isolated data. 
This also added the following features:
- column types are only added, if you visit the "Data Types" tab at least once
- reload possibility of the dataset since it had to be isolated to save the changes from the handsontable edit
- possibility to use similar code to also let the user the column names and not only the types

cheers Ben

ben-schwen committed:
added new convertpossibilites
